dependencies64/ffmpeg/ffmpeg-build-linux/build-ubuntu.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
+#compile ffmpeg successful on ubuntu 14.04.3 & 16.04.1 amd64 using the CasparCG build requirments described here:
+#https://raw.githubusercontent.com/casparcg/Server/2.1.0/BUILDING
 
-sudo apt-get install build-essential curl tar pkg-config
+sudo apt-get -y update
+sudo apt-get -y upgrade
+sudo apt-get -y install build-essential curl tar pkg-config python2.7-dev zlib1g-dev autoconf libtool subversion libfontconfig1-dev ant default-jre default-jdk frei0r-plugins-dev libv4l-dev
 
```

Comment:
Ok, I don't think it is wrong that it is mentioned twice. It makes the ffmpeg-build-linux directory self sustaining in its own, which is good.